### PR TITLE
File Manager - Fix files tree view root folder selection and node expanded state persistence

### PR DESCRIPTION
**Pull request by vladkovl**
Key functionalities:
- fixed root folder selection for files tree view
- fixed folder nodes expanded state persistence for files tree view

### CODE REVIEW

File: js/ui/file_manager/ui.file_manager.breadcrumbs.js [hunk 1]
```diff
@@ -38,76 +38,60 @@ class FileManagerBreadcrumbs extends Widget {
...
-                result.push(item);
-
-                if(i !== parts.length - 1) {
-                    const itemSeparator = {
-                        icon: "spinnext",
-                        isPathSeparator: true
-                    };
-                    result.push(itemSeparator);
-                }
+                result.push(itemSeparator);
```
Comment (jsdmitry) [inline js/ui/file_manager/ui.file_manager.breadcrumbs.js]: May be
```
 result.push({
                    icon: "spinnext",
                    cssClass: FILE_MANAGER_BREADCRUMBS_PATH_SEPARATOR_ITEM_CLASS
                });
```

File: js/ui/file_manager/ui.file_manager.breadcrumbs.js [hunk 2]
```diff
@@ -38,76 +38,60 @@ class FileManagerBreadcrumbs extends Widget {
...
+            const part = parts[i];
+            const partPath = i === 0 ? "" : part;
+            const partText = getName(part);
+
+            const item = {
+                text: partText,
+                path: partPath,
+                isPathItem: true
+            };
+            result.push(item);
```
Comment (jsdmitry) [inline js/ui/file_manager/ui.file_manager.breadcrumbs.js]: May be
```
 result.push({
                text: partText,
                path: partPath,
                isPathItem: true
            });
```

File: js/ui/file_manager/ui.file_manager.breadcrumbs.js (hunk 1 above)
Comment (vladkovl) [reply js/ui/file_manager/ui.file_manager.breadcrumbs.js]: OK

File: js/ui/file_manager/ui.file_manager.breadcrumbs.js (hunk 2 above)
Comment (vladkovl) [reply js/ui/file_manager/ui.file_manager.breadcrumbs.js]: OK

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**